### PR TITLE
Change java.arrays org and module name

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Related issues:
[#1171](https://github.com/ballerina-platform/ballerina-standard-library/issues/1171)

### CODE REVIEW

**Review by daneshk — APPROVED** [review]

**Review by ThisaruGuruge — CHANGES_REQUESTED** [review]
There are places where we special-cased the java.arrays version in `release/src/update_distribution.py`. We have to update that too.

File: release/resources/module_list.json [hunk 1]
```diff
@@ -4,13 +4,14 @@
         "module-ballerina-cache",
         "module-ballerina-crypto",
         "module-ballerina-email",
+        "module-ballerina-encoding",
```
Comment (ThisaruGuruge) [inline release/resources/module_list.json:7]: Why are we adding the encoding package again?

File: release/resources/module_list.json [hunk 2]
```diff
@@ -30,7 +31,6 @@
         "module-ballerina-time",
         "module-ballerinai-transaction",
         "module-ballerina-udp",
-        "module-ballerina-url",
```
Comment (ThisaruGuruge) [inline release/resources/module_list.json:33]: Mistake?

Comment (kalaiyarasiganeshalingam) [reply release/resources/module_list.json]: Yes.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**